tests/test_edgeql_data_migration.py

```diff
@@ -3344,6 +3344,9 @@ async def test_edgeql_migration_reject_prop_05(self):
         await self.fast_forward_describe_migration()
 
     async def test_edgeql_migration_vector_change_01(self):
+        if not self.is_superuser:
```

Comment:
Superuser is not required to create extensions, so we need to adjust our `NOCREATEROLE` mode to include that permission bit instead.  Otherwise we'll be skipping this test on hosted Postgres tests and I'd like to avoid that.

Reply:
Fixed with [sample run](https://github.com/edgedb/edgedb/actions/runs/8008439238)